customer-config/scan-and-go/sng.settings.v1.json

```diff
@@ -10,6 +10,11 @@
       "type": "string",
       "format": "uri",
       "description": "Checkout Engine API URL"
+    },
+    "handoverControlsToSco": {
+      "default": false,
+      "description": "When true, Scan&Go will handover controls to SCO",
```

Comment:
When set to true, Scan&Go bypasses basket controls, assuming they are managed externally.